py3-pythran.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: py3-pythran
+  version: 0.13.1
+  epoch: 0
+  description: Ahead of Time compiler for numeric kernels
+  copyright:
+    - license: BSD 3-Clause
+  dependencies:
+    runtime:
+      - py3-ply
+      - py3-gast
+      - numpy
+      - py3-beniget
+      - python3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python3
+      - py3-setuptools
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/serge-sans-paille/pythran
+      tag: ${{package.version}}
+      expected-commit: 72237f2c3ef4be5e3e1875d4f21d46fa4ac71622
+
+  - name: Python Build
+    runs: python setup.py build
+
+  - name: Python Install
+    runs: python setup.py install --prefix=/usr --root="${{targets.destdir}}"
+
+  - uses: strip
+
+# no versions returned from github graphql api
```

Comment:
we should be able to use github 

```
update:
  ignore-regex-patterns:
    - .post1
  enabled: true
  github:
    identifier: serge-sans-paille/pythran
    use-tag: true
```
validated using `wolfictl check update  --override-version 0.0.1 py3-pythran`